src/components/RelatiesKoppelingen/RelatiesKoppelingen.js

```diff
@@ -133,8 +141,12 @@ const RelatiesKoppelingen = ({
                 })
 
         /**
-         * Function to get and the relations to an object
-         * @returns {Promise} Promise object contains the data from the API
+         * Function to get and return the beleidsrelaties specified on the Naar_Beleidskeuze from an API request using the uuidTo parameter.
+         *
+         * @function
+         *
+         * @param {UUID} uuidTo - Parameter containing a UUID
```

Comment:
{UUID} -> {string}

Reply:
I've changed the parameter type to string.